recipe-box-frontend/src/main/frontend/src/components/Header/Header.css

```diff
@@ -51,6 +55,8 @@
     border-radius: 6px;
     border: 2px solid #fff;
     font-size: 12px;
+    font-weight: bold;
+
```

Comment:
When the viewport gets too narrow, the button gets sliced up, i suggest using `white-space: nowrap;` for it and for Meal Plans.